.circleci/config.yml

```diff
@@ -539,7 +539,32 @@ run_e2e_tests: &run_e2e_tests
         DEV_TAG=${latest/v/}
         IMG_MODIFIER=""
       fi
-      if ./script/e2e-test.sh $USE_MULTICLUSTER_OIDC_ENV $OLM_VERSION $DEV_TAG $IMG_MODIFIER $TEST_TIMEOUT_MINUTES $DEFAULT_DEX_IP $ADDITIONAL_CLUSTER_IP $KAPP_CONTROLLER_VERSION $CHARTMUSEUM_VERSION; then
+      
+      source ./script/lib/liblog.sh
+      info "###############################################################################################"
+      info "USE_MULTICLUSTER_OIDC_ENV: $USE_MULTICLUSTER_OIDC_ENV"
+      info "OLM_VERSION: $OLM_VERSION"
+      info "DEV_TAG: $DEV_TAG"
+      info "IMG_MODIFIER: $IMG_MODIFIER"
+      info "TEST_TIMEOUT_MINUTES: $TEST_TIMEOUT_MINUTES"
+      info "DEFAULT_DEX_IP: $DEFAULT_DEX_IP"
+      info "ADDITIONAL_CLUSTER_IP: $ADDITIONAL_CLUSTER_IP"
+      info "KAPP_CONTROLLER_VERSION: $KAPP_CONTROLLER_VERSION"
+      info "CHARTMUSEUM_VERSION: $CHARTMUSEUM_VERSION"
+      info "###############################################################################################"
```

Comment:
This debug info is gonna help identify some future problems and mismatches. Thanks!